.github/workflows/main.yml

```
@@ -0,0 +1,62 @@
+name: "CI"
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+env:
+  CI: true
+
+jobs:
+  Test:
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        channel: [stable, beta]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: UziTech/action-setup-atom@v1
+      with:
+        channel: ${{ matrix.channel }}
+    - name: Atom version
+      run: atom -v
+    - name: APM version
+      run: apm -v
+    - name: Install dependencies
+      run: apm ci
+    - name: Run tests 👩🏾‍💻
+      run: atom --test spec
+  Lint:
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: NPM install
+        run: npm ci
+      - name: Lint ✨
+        run: npm run lint
+
+  Release:
+    needs: [Test, Lint]
+    if: github.ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: UziTech/action-setup-atom@v1
+      - name: NPM install
+        run: npm ci
+      - name: Release 🎉
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          ATOM_ACCESS_TOKEN: ${{ secrets.ATOM_ACCESS_TOKEN }}
+        run: npm run semantic-release
```

This isn't a package script, but a utility that gets ran.
```suggestion
        run: npx semantic-release
```
See their [example Action](https://github.com/semantic-release/semantic-release/blob/master/docs/recipes/github-actions.md) for details, but the rest of this looks fine.

---

I added it as a package script. That way it uses the version of `semantic-release` in `package.json` not always the latest in case we need to hold it back for some reason.

---

`npx` uses the version installed in the current project, only downloading and installing the latest if it's not in the current project.

I'd rather not have this as a package script since it isn't really intended to be ran manually.

---

I didn't know npx used local packages. I changed it